resources/assets/coffee/react/contest/voting/entry.coffee

```diff
@@ -65,9 +65,9 @@ class Contest.Voting.Entry extends React.Component
       if @props.contest.show_votes
         if @props.contest.best_of
           div className:'contest__vote-count contest__vote-count--no-percentages',
-            "#{@props.entry.results.votes} points"
+            "#{@props.entry.results.votes}", osu.trans("contest.vote.points")
         else
           div className:'contest__vote-count',
-            "#{@props.entry.results.votes} votes"
+            "#{@props.entry.results.votes}", osu.trans("contest.vote.votes")
```

Comment:
This renders without a space between the text.
Also there's a `osu.transChoice 'contest.vote.count', @props.entry.results.votes` for this.